crates/wasi-log/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+    name    = "wasi-log"
+    version = "0.1.0"
+    edition = "2021"
+    authors = [ "Nitish Malhotra <nitishm@microsoft.com>" ]
+
+[lib]
+    crate-type = [ "cdylib" ]
+
+[dependencies]
+    anyhow           = "1.0"
+    wit-bindgen-rust = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "dab589f579b980c81b39c8348400b88a8c4d175a" }
```

Comment:
Could you please rebase on top of https://github.com/fermyon/wasi-experimental-toolkit/pull/5?

Reply:
Will do